app/models/session_form.rb

```diff
@@ -0,0 +1,25 @@
+class SessionForm
+    include ActiveModel::Model
+  
+    attr_accessor(
+      :email,
+      :password
+    )
+  
+    validates :email, presence: true, format: { with: /\A\S+@.+\.\S+\z/ }
+    validates :password, presence: true
+    validate :user_valid?
+  
+    def user
+      User.find_by(email: email)
```

Comment:
it's better to use smth like
`@user ||= User.find_by(email: email)`
otherwise you will call DB each _user_ call